### PR TITLE
Add support for the select operator.

### PR DESCRIPTION
**Pull request by realityking**
Thanks to @Haehnchen for the initial contribution in #41.

### CODE REVIEW

File: tests/E2E/EntrySelectTest.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        $entries = $this->client->getEntries($query);
+
+        $this->assertInstanceOf(ResourceArray::class, $entries);
+        $this->assertEquals(null, $entries[0]->getName());
+    }
+
+    /**
+     *
+     */
+    public function testSelectOnlyOneField()
```
Comment (Haehnchen) [inline tests/E2E/EntrySelectTest.php]: empty comment

File: tests/E2E/EntrySelectTest.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    public function setUp()
+    {
+        $this->client = new Client('b4c0n73n7fu1', 'cfexampleapi');
+    }
+
+    /**
+     *
+     */
+    public function testSelectOnlyMetatdata()
```
Comment (Haehnchen) [inline tests/E2E/EntrySelectTest.php]: empty comment

**Comment by Haehnchen** [comment]
thx. good idea about `sys` filtering

File: tests/E2E/EntrySelectTest.php (hunk 2 above)
Comment (realityking) [inline tests/E2E/EntrySelectTest.php]: Thanks, forgot to add them again.

File: tests/E2E/EntrySelectTest.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        $query = (new Query)
+            ->setContentType('cat')
+            ->select(['fields.name'])
+            ->where('sys.id', 'nyancat')
+            ->setLimit(1);
+        $entries = $this->client->getEntries($query);
+
+        $this->assertInstanceOf(ResourceArray::class, $entries);
+        $this->assertEquals('Nyan Cat', $entries[0]->getName());
```
Comment (dlitvakb) [inline tests/E2E/EntrySelectTest.php]: I would add another test checking that only `name` got sent and the rest of the fields are empty.
